docs/guides/integration-examples/test-runners.md

```diff
@@ -142,3 +142,66 @@ For AVA there is a [detailed written tutorial](https://github.com/zellwk/ava/blo
 :::note
 Note that this tutorial is pre mongodb-memory-server 7.x.
 :::
+
+## vitest
+
+For [vitest](https://vitest.dev/), create a [global setup file](https://vitest.dev/config/#globalsetup).
```

Comment:
Please add a badge similar to jest and mocha:
https://github.com/typegoose/mongodb-memory-server/blob/e4992fd4f89d5025d315d05d7ca6d43358cf62be/docs/guides/integration-examples/test-runners.md?plain=1#L10